src/edu/stanford/nlp/dcoref/CoNLL2011DocumentReader.java

```diff
@@ -971,7 +971,7 @@ public static void main(String[] args) throws IOException {
       System.exit(-1);
     }
     PrintWriter fout = new PrintWriter(outfile);
-    logger.info("Writing to " + outfile);
+    logger.finest("Writing to " + outfile);
```

Comment:
This is a pretty good counter-example to your hypothesis...

Reply:
Thanks for the feedback @kno10. I believe our hypothesis may only apply to a certain class of logs and not all logs in general. Nevertheless, we are [working on making the transformations a bit less drastic](https://github.com/ponder-lab/Logging-Level-Evolution-Plugin/issues/232).